example/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:4.4.4
```

Comment:
Consider bumping to 6.6.0 to retain parity with our production environment. If you do, don't forget to add a test for 6.6.0 in `.travis.yml`